init-tools.sh

```diff
@@ -164,6 +164,8 @@ if [ ! -e $__INIT_TOOLS_DONE_MARKER ]; then
     echo Copying supplemental overrides from Tools-Override.
     cp $__scriptpath/Tools-Override/* $__scriptpath/Tools
 
+    Tools-Override/crossgen.sh $__scriptpath/Tools
```

Comment:
Could you add a condition for a way to disable this (I think I might want the flexibility to opt-out for some of the stuff I'm working on.  I had made a similar change to Roslyn's crossgen script at some point).